src/__tests__/initRules.test.js

```diff
@@ -1776,7 +1776,7 @@ describe('initRules', function() {
         expect(logger.warn.calls.count()).toBe(1);
       });
 
-      it('logs a message when a rule completes', function() {
+      it('logs a message when a rule completes', function(done) {
```

Comment:
We should really just do `return lastPromiseInQueue.then(function() { ...` instead of using `done`, but this is fine.